Content.Server/GameObjects/EntitySystems/MoverSystem.cs

```diff
@@ -131,16 +132,16 @@ public override void Update(float frameTime)
                 var physics = entity.GetComponent<PhysicsComponent>();
                 if (entity.TryGetComponent<CollidableComponent>(out var collider))
                 {
-                    UpdateKinematics(entity.Transform, mover, physics, collider);
+                    UpdateKinematics(entity.Transform, mover, physics, frameTime, collider);
                 }
                 else
                 {
-                    UpdateKinematics(entity.Transform, mover, physics);
+                    UpdateKinematics(entity.Transform, mover, physics, frameTime);
                 }
             }
         }
 
-        private void UpdateKinematics(ITransformComponent transform, IMoverComponent mover, PhysicsComponent physics, CollidableComponent collider = null)
+        private void UpdateKinematics(ITransformComponent transform, IMoverComponent mover, PhysicsComponent physics, float frameTime, CollidableComponent collider = null)
```

Comment:
Note; this isn't used, but the idea is that maybe at some point it _will be used_ in preference over adding an IGameTiming dependency.